Gemfile

```diff
@@ -18,3 +18,6 @@ gem 'wdm', '>= 0.1.1' if Gem.win_platform?
 gem "tzinfo-data", platforms: [:mingw, :mswin, :x64_mingw, :jruby]
 
 # For what ~> means, see https://robots.thoughtbot.com/rubys-pessimistic-operator
+
+# Update Kramdown parser, see https://stackoverflow.com/questions/63335953/jekyll-error-building-page-related-to-kramdown-parser
```

Comment:
@LaurenEllwood Thanks, I hadn't seen this change. While this SO post provides the fix, the official Jekyll docs (which I've only just seen now) are probably a better explanation for the underlying issue: https://jekyllrb.com/news/2020/08/05/jekyll-3-9-0-released/

Can you add that URL instead, please?